worker/src/primary_connector.rs

```diff
@@ -42,17 +42,15 @@ impl PrimaryConnector {
 #[derive(Clone)]
 pub struct PrimaryClient {
     /// The primary network address.
-    _primary_address: SocketAddr,
+    _primary_address: Multiaddr,
     client: WorkerToPrimaryClient<Channel>,
 }
 
 impl PrimaryClient {
-    pub fn new(address: SocketAddr) -> Self {
-        // TODO maybe use TLS
```

Comment:
Ditto: I'd like to not lose this comment, because in the mTLS context, using TLS is not just about a change of protocol, but requires configuring a custom verifier.